Including both linux/stat.h and sys/stat.h results in redefinitionso struct
statx{,timestamp}.
--- a/io_uring_engine.h
+++ b/io_uring_engine.h
@@ -7,7 +7,7 @@
 #include <string_view>
 #include <sys/socket.h>
 #include <sys/types.h>
-#include <linux/stat.h>
+#include <sys/stat.h>
 
 struct io_uring_sqe;
 #ifndef WITHOUT_URING
